tooling/ssa_fuzzer/fuzzer/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "fuzz"
```

Comment:
We might end up with having other fuzzers in the project, at which point this name will have to be a bit more specific I think.